### ISSUE TITLE
[7.0.3RC2] Local files still encrypted after sync download

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#11908]
### Steps to reproduce:
1. Not sure, but I'm recovering from the situation raised here: https://github.com/owncloud/core/issues/11906 where all etags changed.

### Expected result
All files properly downloaded

### Actual result
Some files locally encrypted, but fine on server.

One file, "articles.txt" looks like this:
```
HBEGIN:cipher:AES-256-CFB:HEND-----
```

Another file "pomodoro.txt" (which is my TODO list) has a conflict:
The file "pomodoro.txt" is still encrypted.
The conflict file "pomodoro_conflict-20141101-091042.txt" is fine.

If I download "articles.txt" and "pomodoro.txt" from the web UI the files are decrypted properly.

VERY STRANGE.

### Versions
ownCloud 7.0.3 RC2
Sync client 1.7.0 beta1

CC @schiesbn 

@karlitschek @MTRichards @craigpg potential showstopper. Need to find a way to reproduce it locally first (only happened on my own server so far)

**Comment by sjlawson** [comment]
This is happening to me, but I'm not the administrator of the server. Using client ver 1.5.0 on Ubuntu 14.04. The server uname -a output is: 
Linux ---.---.org 2.6.32-504.8.1.el6.x86_64 #1 SMP Wed Jan 28 21:11:36 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
OwnCloud server ver 8.0

Is there a way to manually decrypt the files?

**Comment by PVince81** [comment]
I believe we fixed this a while ago and it was due to cookie handling ? @LukasReschke @javiergonzper @schiesbn 

@sjlawson you cannot decrypt the file locally. To recover from this situation, either:
A) Reset your local sync client and let it redownload everything
or
B) Setup a separate sync client and let it download everything (or use selective sync to only focus on specific directories). Make sure that the download files are not encrypted. Then "touch" the files you knew to be broken so that the second sync client reuploads these files. This will make the first sync client think the files changed and it will redownload them. At least that's how I recovered mine.

Advanced stuff: what I did was `grep -rl HBEGIN *` which gave me the list of "broken" files. Then in the folder of the second client, I triggered "touch" for every file in that list.
